api/parts/mgmt/ip.js

```diff
@@ -8,7 +8,8 @@
 var ip = {},
     net = require('net'),
     extIP = require('external-ip'),
-    plugins = require('../../../plugins/pluginManager.js');
+    plugins = require('../../../plugins/pluginManager.js'),
+    offlineMode = plugins.getConfig("api").offline_mode;
```

Comment:
Here would be the same case
you don't need to call `loadConfigs`, but you would need to reread configs using `getConfig` on each getHost function call, not once per file.